ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-matrix-answer/activity-matrix-answer.component.html

```diff
@@ -0,0 +1,82 @@
+<ddp-question-prompt [block]="block"></ddp-question-prompt>
+
+<div class="table-container">
+  <table class="matrix-answer-table" data-test-id="matrix-answer-table">
+    <thead class="matrix-answer-table__header">
+      <tr
+        *ngIf="renderGroups.length > 1"
+        class="matrix-answer-table__row matrix-answer-table__row--header matrix-answer-table__row--groups"
+        data-test-id="table-header-groups"
+      >
+        <th class="matrix-answer-table__cell matrix-answer-table__cell--header">&nbsp;</th>
+        <th
```

Comment:
I've added a lot of different classes with modifiers it would be easy to customise the look of this question in each study. I've only added a handful of common styles which can be easily overridden.